dashboard/app/helpers/levels_helper.rb

```diff
@@ -594,4 +594,9 @@ def can_view_solution?
       Ability.new(current_user).can? :view_level_solutions, @script
     end
   end
+
+  def should_show_inline_answer?
```

Comment:
Is this a property of the `Level`?  `ScriptLevel`?  `User`?  Prefer it on the model, e.g.:

```ruby
current_user.can_see_inline_answer? @script_level
# or
@script_level.should_show_inline_answer_to current_user
```